core/reference_data.py

```diff
@@ -59,3 +59,41 @@ def seed_geospatial_dim_table():
                 db.session.add(geospatial_record)
 
     db.session.commit()
+
+
+def seed_fund_table():
+    """
+    Input seed data to fund_dim table using the mappings in /tests/resources/fund_dim.csv.
```

Comment:
Maybe something in here saying that this is for local dev only. Obviously not just this function, but it looks slightly weird that this is in the `core` of the app but then reaches into some test data fixture to insert into the DB. I think just being super clear this isn't intended to touch any of the deployed envs would be nice.